project/.github/stale.yml

```diff
@@ -0,0 +1,16 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
```

Comment:
Keep in mind, this will close nearly all issues...

Reply:
Maybe we should start with a higher value first... 180 days?

Reply:
Yes why not